### ISSUE TITLE
[React Native Gallery -> Permissions]: Keyboard focus and Narrator focus are not on sync when user navigate to 'Request' button. 

### ISSUE THREAD

**Issue by AgneLukoseviciute** [microsoft/react-native-gallery#313]
**Repro-Steps:**
Launch the React Native Gallery app.
Navigate to ‘Permissions’ button and invoke it.
Navigate to ‘Request’ button and invoke it.
Now renavigate to the invoked Request button and observe the keyboard and narrator focus.

**Actual Result:** 
While renavigating to “Request” button narrator focus is not moving with kb focus hence these controls are not announced by the narrator. Narrator focus stays on the last interacted element.
 
**Expected Result:**
While navigating to “Request” button narrator should announce the proper information to the user as “Request windows.permission.appCaptureServices,  button, disabled”.
 
**User Impact:**
Users with visual impairment who rely on screen reader will get confused if narrator focus and Keyboard focus are not sync.

Internal bug link: https://microsoft.visualstudio.com/DefaultCollection/OS/_workitems/edit/43361620

**Comment by chrisglein** [comment]
Perhaps related to #312
